src/bugreporter/odemis_bugreporter.py

```diff
@@ -0,0 +1,542 @@
+        button_sizer.Add(cancel_btn, 0, wx.ALL, 10)
+        report_btn = wx.Button(panel, wx.ID_ANY, "Report")
+        report_btn.Bind(wx.EVT_BUTTON, self.on_report_btn)
+        button_sizer.Add(report_btn, 0, wx.ALL, 10)
+
+        # Status update label
+        # TODO: replace by an animated throbber
+        wait_lbl = wx.StaticText(panel, wx.ID_ANY, "")
+        sizer.Add(wait_lbl, 10, wx.EXPAND | wx.ALL, 10)
+        sizer.Add(button_sizer, 0, wx.ALIGN_RIGHT | wx.ALL, 10)
+
+        panel.SetSizer(sizer)
+
+        self.Bind(wx.EVT_CLOSE, self.on_close)
+        self.Centre()
+        self.Show()
+        self.Layout()
+
+        # Make these elements class attributes to easily access the contents
+        self.panel = panel
+        self.name_ctrl = name_ctrl
+        self.email_ctrl = email_ctrl
+        self.summary_ctrl = summary_ctrl
+        self.description_ctrl = description_ctrl
+        self.wait_lbl = wait_lbl
+
+        self.bugreporter = controller
+
+        # flag is set to False when the backspace is pressed
+        self.make_suggestion = True
+
+        # Load known users, if not available make config file
+        self.known_user_parser = ConfigParser.ConfigParser(dict_type=OrderedDict)
+        conf_path = os.path.join(os.path.expanduser(u"~"), '.config', 'odemis', 'bugreporter.config')
+        with open(conf_path, 'a+') as f:
+            self.known_user_parser.readfp(f)
+        if 'users' not in self.known_user_parser.sections():
+            self.known_user_parser.add_section('users')
+
+    def store_user_info(self, name, email):
+        """
+        Store the user name and email in the config file, so it can be suggested the
+        next time the bugreporter is used.
+        :arg name: (String) user name
+        :arg email: (String) user email
+        """
+        # Add user to top of config file, truncate file if it contains too many users.
+        # Adding the user to the top of the list ensures that the suggestion is made based
+        # on the latest entry. Otherwise, if a typo occurred the first time the name was written,
+        # the faulty name will always be suggested.
+        if len(self.known_user_parser.items('users')) >= MAX_USERS:
+            oldest_entry = self.known_user_parser.items('users')[-1][0]
+            self.known_user_parser.remove_option('users', oldest_entry)
+        # It's not very good coding practice to manipulate ._section, but there is no
+        # other option in ConfigParser to add the new item at the first position.
+        # It would also be nicer not to create a new ordered dictionary, but to move the
+        # element internally. The python 3 version has such a function (move_to_end).
+        self.known_user_parser.remove_option('users', name)
+        prev_items = self.known_user_parser._sections['users'].items()
+        self.known_user_parser._sections['users'] = OrderedDict([(name, email)] + prev_items)
```

Comment:
Check that any character not supported in the config file is removed (or that every character is supported).

Reply:
it should now support all characters